README.md

```diff
@@ -165,7 +165,7 @@ You can control the power on a USB port(s) like this:
     uhubctl -a off -p 2
 
 This means operate on default smart hub and turn power off (`-a off`, or `-a 0`)
-on port 2 (`-p 2`). Supported actions are `off`/`on`/`cycle` (or `0`/`1`/`2`).
+on port 2 (`-p 2`). Supported actions are `off`/`on`/`cycle`/`toggle` (or `0`/`1`/`2`/`3`).
```

Comment:
Please add toggle mention to `print_usage()` in `uhubctl.c`

Reply:
Of course, yes.